ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/client/auth/JwtSecretKeyLoader.java

```diff
@@ -35,6 +35,7 @@
   private static final Logger LOG = LogManager.getLogger();
 
   public static final String JWT_SECRET_FILE_NAME = "ee-jwt-secret.hex";
+  public static final String UNPREFIXED_JWT_SECRET_FILE_NAME = "unprefixed-ee-jwt-secret.hex";
```

Comment:
I think my first comments misread this PR - we only seem to use this constant from the test and it does make sense to write out a test file without the prefix so should just move this to the test instead of production code.